app/src/main/java/app/revanced/manager/ui/screen/settings/AboutSettingsScreen.kt

```diff
@@ -152,23 +182,20 @@ fun AboutSettingsScreen(
                         }
                     }
                 }
-                outlinedButton.forEach { (icon, text, onClick) ->
-                    OutlinedButton(
-                        onClick = onClick
-                    ) {
-                        Row(
-                            horizontalArrangement = Arrangement.spacedBy(8.dp),
-                            verticalAlignment = Alignment.CenterVertically
+                FlowRow(
+                    modifier = Modifier.padding(top = 12.dp),
+                    horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally)
+                ) {
+                    socialButtons.forEach { (icon, onClick) ->
+                        IconButton(
+                            onClick = onClick,
+                            modifier = Modifier.padding(end = 8.dp),
                         ) {
                             Icon(
                                 icon,
                                 contentDescription = null,
-                                modifier = Modifier.size(18.dp)
-                            )
-                            Text(
-                                text,
-                                style = MaterialTheme.typography.labelLarge,
-                                color = MaterialTheme.colorScheme.primary
+                                modifier = Modifier.size(28.dp),
+                                tint = MaterialTheme.colorScheme.secondary
                             )
                         }
                     }
```

Comment:
Please move this out of the FlowRow, any new entry added in the API will cause this to break